packages/core/src/utilities/getTargetVolumeAndSpacingInNormalDir.ts

```diff
@@ -74,9 +78,12 @@ export default function getTargetVolumeAndSpacingInNormalDir(
       viewPlaneNormal
     );
 
-    if (spacingInNormalDirection < smallest.spacingInNormalDirection) {
+    // Allow for 1/1000 part larger requirement to prefer earlier volumes
+    // when the spacing is all essentially the same.
+    if (spacingInNormalDirection * 1.001 < smallest.spacingInNormalDirection) {
```

Comment:
How about 

```
if (Math.abs(spacingInNormalDirection - smallest.spacingInNormalDirection) > CORE.Constant.Epsilon)
```

Reply:
No, I don't think that is as good, this spacing check is related to the size of the actual spacing, and is multiplicative rather than absolute, and suppose we had extremely fine spacing, then using a fixed constant will either be too big or too small.  I'm ok with  (spacingInNormalDirection * (1+CORE.Constant.Epsilon)) if you want?